python/activity_stream/activity_stream.py

```diff
@@ -412,6 +496,39 @@ def load_data(self, sg_entity_dict):
         self._bundle.log_debug("Ask db manager to ask shotgun for updates...")
         self._data_manager.rescan()
         self._bundle.log_debug("...done")
+
+    def show_new_note_dialog(self, modal=True):
+        """
+        Shows a dialog that allows the user to input a new note.
+
+        :param modal:   Whether the dialog should be shown modally or not.
+        :type modal:    bool
+        """
+        note_dialog = NoteInputDialog(parent=self)
+        note_dialog.entity_created.connect(self._on_entity_created)
+        note_dialog.data_updated.connect(self.rescan)
+        note_dialog.set_bg_task_manager(self._task_manager)
+        note_dialog.set_current_entity(self._entity_type, self._entity_id)
+
+        if modal:
+            note_dialog.exec_()
```

Comment:
Ah, so we are in a holding pattern now, and won't release these changes until the dynamite integration is released? So that there can be tweaks and bug fixes? And then, the next step is to upgrade RV to use these official widgets instead? That sounds cool!